docs/2023-06-08-meeting-transcript.md

```diff
@@ -0,0 +1,881 @@
+
+One or more of their team memebers must've been compromised and the attacker has used the login information to upload further instances of the virus. 
+
+Because of the quick response, the team was able to get the malicious server nullrouted, and also prompted attacker to temporarily disable the Stage 2 and its attempt to thwart analysis. 
+
+In the investigation, it was revealed that Fracturiser was presented on mods uploaded as early as April 18th of this year. 
+
+Accroding to CurseForge, 6000 unique users downloaded copies of the virus. 
+
+Because of the popularity of the Lunar Pixel projects, if it was not caught as quickly as was on the 7th, 24 hours could have translated to millions of infected users. 
+
+The purpose of today's meeting is to discuss what to do next. The Fractureiser Mitigation Team has successfully prevented further spread of the virus, and has worked on further detection, mitigation efforts with the community.
+
+And the most dangerous period of it has now passed. But it's necessary to reflect on how easy it was for mailicious actor to pull off attack of scale, and how close they got to significant numbers. 
+
+Today's topic: how do we stop another Fractureiser before it's written?
+
+We have an agenda of five (5) topics today, and I'm gonna hand it off to willie to introduce the first topic. 
+
+## Topic 1: Opaque Review Processes/Security by Obscurity
+
+[Start at 00:02:24]
+
+Williewillus: Alright. Hey everyone, I'm willie, and I've kind of been serving as the secretary of this whole thing.
+
+I kinda wrote up this agenda, that you, uh that is linked in the GitHub, and I guess, I've ordered these topics, 
+there's about 4 or 5 of them, from the most realistic or actionable to the most kinda like, "pie in the sky". 
+
+So if we run out of time, we'll probably cut some of the later ones. 
+
+The first one I'd like to start with, I guess, just from the perspective of the community or the incident response, was the questions like what actually happens when a mod repository "reviews" the mod that is submitted. Are there new mod or an update? And what sort of checks are being run, what should be put in place, and those are sort of the question that I'd like to discuss in the first point. 
+
+Back to you, Emi or Jasmine.
+
+Emi: sure, it looks like Fury has requested to speak, so we'll give them the floor. 
+
+Fury: Hi everybody. My name is Fury, and, CEO of Overwolf. Emi, before I touch on this uh first topic, um sorry willie. 
+
+I just want to say, we're... I'm very grateful, we're all very grateful in Overwolf and CurseForge, for all the help we received from a lot of the folks here, 
+
+Super ??? in my opinion, that there is such involvement, dedication and commitment on behalf of the community. The way the things happened, I agree that if this is not caught on time, the impact could've been much more severe.  
```

Comment:
I also have no idea what they're saying here.

Reply:
> I don't have the audio, but I'd say "nice", "cool" or "awesome" would fit

Quote the doc:

> You can watch the recording on [YouTube](https://www.youtube.com/watch?v=L52Hu334Q90) or [PeerTube](https://tube.sleeping.town/w/c48e7df1-cf9b-43d2-84a8-4bce404ee836).

Reply:
sounds like "intriguing" to me

Reply:
imo sounds like the word ends in -ling. Why don't we ping those people to decipher their own ~~writing~~ speech?